Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -112,6 +112,14 @@ function vtkMouseRangeManipulator(publicAPI, model) {
   //-------------------------------------------------------------------------
   publicAPI.onButtonDown = (interactor, renderer, position) => {
     model.previousPosition = position;
+    const glRenderWindow = interactor.getView();
+    // Ratio is the dom size vs renderwindow size
+    const ratio =
+      glRenderWindow.getContainerSize()[0] / glRenderWindow.getSize()[0];
```

Comment:
Just curious how the container and the render window could have a different ratio? Is the mouse event position scaled by that ratio also? Can't we use the container size right away?

Reply:
The mouse will be screen-based, but yes the container and the window can have some weird ratio. So no you can not use the container right away.